### ISSUE TITLE
Script Generator: Can't use packaged git distribution

### ISSUE THREAD

**Issue by Alistair-McGann-Tessella** [ISISComputingGroup/IBEX#5700]
As a user of the script generator I don't want to be required to install git. It has been reported that when running the standalone script generator without having git already installed on the system that the script generator fails to load.

After merging this ticket, we should consider making a patch release to the script generator to address this.

## Acceptance criteria
- [ ] When using the standalone script generator without having a system git installation, the script generator can use git functions

**Comment by Alistair-McGann-Tessella** [comment]
PR: https://github.com/ISISComputingGroup/ibex_gui/pull/1188

**Comment by JamesKingWork** [comment]
I think we should definitely make a patch release for this otherwise the script generator will be effectively unusable for many until the next release

**Comment by kjwoodsISIS** [comment]
Yes, absolutely.  We need this issue resolved asap.

**Comment by JamesKingWork** [comment]
# Script Generator VM Testing

https://developer.microsoft.com/en-us/windows/downloads/virtual-machines/

Unzip download and open ova file with virtualbox.

Create shared folder to put script generator in. Auto mount and make permanent selected.

Put built script generator into shared folder then copy from there onto VM (Doesn't like running from shared folder and drag and drop feature regularly fails).

Then run and test whatever you want.

Things to check:

- You are using the bundled python
  - Start task manager and look at the python running the script generator script
- You are using the bundled JRE
- You are using the bundled git

## Licensing

- https://windowsdev.azureedge.net/eulas/WindowsDeveloperVirtualMachineEula20151118.pdf
- Section 7 describes what we are not allowed to use it for
  - This does not include any of our currently intended purposes
- Section 1 part c worries me slightly
  - I think we are using it to demonstrate and internally evaluate software on windows
  - I am not sure whether or not this would be classed as a commercial purpose (I think this means as commercial hosting so I believe it is fine if you take into account Exhibit A 2cv)
- Section 4
  - We can have a backup copy and reinstall it
  - Does this mean we can keep reinstalling it?
- Exhibit A 13dii "you may not use it after the evaluation period"
  - Does this mean that we cannot reinstall it or use the same VM?

**Comment by Alistair-McGann-Tessella** [comment]
Test on VM highlighted another problem with missing visual studio c++ redistributables. This was also seen in https://github.com/ISISComputingGroup/IBEX/issues/5682.

Solution was to copy the VC redist files from ndw1757 into the directory containing the python executable. I've copied these files to the zip on the share containing the genie_python dependencies. Using these DLLs, everything worked on the blank VM with no extra installation required.

Merging this, will create a release of the script generator on Monday after the builds have happened

**Comment by Alistair-McGann-Tessella** [comment]
Because there is another ticket (#5708) which needs to be merged before the release can be made, the creation of the release has been moved to #5730 and this can be closed
